.travis.yml

```diff
@@ -6,11 +6,17 @@ node_js:
 
 matrix:
   fast_finish: true
+  include:
+    - node_js: 7 # Version used to deploy to npm registry
+      env: BUILD_ENV=production
 
```

Comment:
No, otherwise `devDependencies` aren't installed. The rationale is: run `npm install` in non-production environment to be able to build, but build for production environments at release time.